hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/admin/om/ListOpenFilesSubCommand.java

```diff
@@ -66,6 +66,11 @@ public class ListOpenFilesSubCommand implements Callable<Void> {
       description = "Format output as JSON")
   private boolean json;
 
+  @CommandLine.Option(names = { "--hide-deleted" },
+      defaultValue = "false",
```

Comment:
I would suggest we hide the deleted by default.

Reply:
Maybe we can call this --show-deleted.

Reply:
Done